### ISSUE TITLE
Audio stream

### ISSUE THREAD

**Issue by MaazAhmadDeveloper** [piyushgarg-dev/React-webRTC#1]
piyush bro in this we can only stream video but it's not a complete tutorial tell us how can we stream audio with a video call ?

**Comment by SDE-ADNAN** [comment]
yes absolutely pls share something  @piyushgarg-dev  for this , btw love your videos bro

**Comment by virajhapaliya** [comment]
Have you tried with removing "muted" from <ReactPlayer> element in remote stream ?
